upgrade/1.2.6/README.md

```diff
@@ -31,14 +31,14 @@ Earlier version of CSM must first be upgraded to at least `v1.2.0`. For informat
    **IMPORTANT**: If necessary, change this command to match the actual location of the extracted files.
 
    ```bash
-   ncn-m001# CSM_DISTDIR="$(pwd)/csm-1.2.6"
+   ncn-m001# export CSM_DISTDIR="$(pwd)/csm-1.2.6"
```

Comment:
Why are we exporting this instead of just setting it? I don't see the variable being used in any of our update scripts.

Reply:
good catch, i used it at one point in the new script but later removed it. I will remove this export